starter-code/js/index.js

```diff
@@ -50,3 +52,35 @@ window.onload = function(){
     deleteButtons[i].onclick = deleteItem;
   }
 };
+*/
+
+function main () {
+
+var costArray = document.querySelectorAll(".cost");
+var quantityArray = document.querySelectorAll(".quantity");
+var totalPrice = 0;
+
+
+var calculateButton = document.getElementById("calculate-btn");
+var totalAmmount = document.querySelector(".total");
+
+  var calculatePrice = function() {
+      for (var i = 0; i < quantityArray.length; i++) {
+        totalPrice+= costArray[i] * quantityArray[i];
+      }
+      // calculateButton.removeEventListener
+      console.log("monkeyBusiness");
+     totalAmmount.innerHTML = totalPrice;
```

Comment:
I think that what you want to do is to update all total amounts. Then what you have to do is a .querySelectorAll('.total') and update them with a for that iterates the quantity and the totalprice to assign.

Reply:
Once you do this you can get all the prices and calculate the sum of all prices